src/test/regress/expected/isolation_select_vs_all.out

```diff
@@ -308,7 +308,7 @@ step s1-commit: COMMIT;
 step s1-select-count: SELECT COUNT(*) FROM select_append;
 count
 
-5
+10
```

Comment:
Investigating why this change happened. Similar to https://github.com/citusdata/citus/pull/4207/files#r497480623